cli/src/cargo.rs

```diff
@@ -31,11 +31,6 @@ pub fn invoke(args: &Args) -> Result<()> {
         } else {
             Stdio::piped()
         })
-        .stderr(if args.no_capture {
-            Stdio::inherit()
-        } else {
-            Stdio::piped()
-        })
```

Comment:
This would capture this warning

![image](https://github.com/Aleph-Alpha/ts-rs/assets/131818645/7a849260-b68b-40ac-9407-79c6b638dfbe)